ddtrace/contrib/botocore/patch.py

```diff
@@ -37,8 +37,8 @@
 config._add(
     "botocore",
     {
-        "distributed_tracing": get_env("botocore", "distributed_tracing", default=True),
-        "invoke_with_legacy_context": get_env("botocore", "invoke_with_legacy_context", default=False),
+        "distributed_tracing": os.getenv("DD_BOTOCORE_DISTRIBUTED_TRACING", default=True),
```

Comment:
Might need to update from `master`, there was a bug and we need `asbool` on these.

Reply:
Is there a special git way I should update from master? Or is it ok to just add asbool here?

Reply:
we'll want to do a PR merging `master` into the `1.0-dev` branch (but we can sync with @Kyle-Verhoog quick to see how they have been doing it, we might not need to do a PR)

Reply:
yeah i relatively recently rebased 1.0-dev off of master. We should probably just merge master in now periodically rather than rebasing. WDYT @brettlangdon

Reply:
rebasing is probably better, higher likelihood of catching issues when someone updates a file that has been changed on `1.0-dev`.

not a strong feeling, whatever means we do it often enough and it is correct, then it is fine.